Dockerfile.efdtransform

```diff
@@ -0,0 +1,46 @@
+# Define the LSST version
+ARG OBS_LSST_VERSION=w_2024_47
+FROM lsstsqre/centos:7-stack-lsst_distrib-${OBS_LSST_VERSION}
```

Comment:
Should change now to `ghcr.io/lsst/scipipe:al9-${OBS_LSST_VERSION}`, with the version at `w_2025_11` or so.